indexer/migration/src/lib.rs

```diff
@@ -17,6 +17,7 @@ mod m20220528_000012_create_plutus_data_table;
 mod m20220808_000013_create_transaction_reference_input_table;
 mod m20221031_000014_create_dex_table;
 mod m20230223_000015_modify_block_table;
+mod m20230927_231206_create_stake_delegation_table;
```

Comment:
let's probably keep the naming convention (date + number)?

Reply:
good catch, I didn't notice that. The weird thing is that I created this file with sea-orm-cli, so I'm not sure why it's choosing that name